<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerDeviceConfigWizardGeneralPage</class>
 <widget class="QWizardPage" name="Mer::Internal::MerDeviceConfigWizardGeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="configurationLabel">
     <property name="text">
      <string>Configuration:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="virtualMachineRadioButton">
       <property name="text">
        <string>Virtual Machine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="realDeviceRadioButton">
       <property name="text">
        <string>Real device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Device name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLineEdit" name="deviceNameLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="virtualMachineComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="architecture">
     <property name="text">
      <string>Architecture:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="architectureLabel">
     <property name="text">
      <string>undefined</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hostNameLabel">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="hostNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="usernameLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="authTypeLabel">
     <property name="text">
      <string>Authentication type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="passwordRadioButton">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="keyRadioButton">
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="SshKeyLabel">
     <property name="text">
      <string>Ssh Key:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="Utils::PathChooser" name="SshKeyPathChooser" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Crete New...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="sshPortLabel">
     <property name="text">
      <string>SSH port:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="sshPortSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="Timeout">
     <property name="text">
      <string>Timeout:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="timeoutSpinBox"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="freePortsLabel">
     <property name="text">
      <string>Free ports:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="freePortsLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="freePortsDebuggerMessageLabel">
     <property name="text">
      <string>Note: For debugging, at least 2 free ports are required.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
